src/components/play/PatternSequencer.js

```diff
@@ -124,7 +124,7 @@ class PatternSequencer extends Component {
                     ))}
 
                 </Box>
-                { (!_.isNil(this.props.user) && !_.isNil(this.props.round) && !_.isNil(this.props.round.userPatterns[this.props.user.id])) &&
+                {(!_.isNil(this.props.user) && !_.isNil(this.props.round) && !_.isNil(this.props.round.userPatterns[this.props.user.id])) &&
                     <>
```

Comment:
what's this?

Reply:
The switch should only be displayed if a user, round and userPattern props are loaded.